README.md

```diff
@@ -299,6 +357,28 @@ spec:
   runtime_assets:
   - nixwiz/http-checks
 ```
+#### http-get
+
+```yml
+---
+type: CheckConfig
+api_version: core/v2
+metadata:
+  name: prometheus-node-exporter
+  namespace: default
+spec:
+  command: http-get --url http://localhost:9100
+  subscriptions:
+  - system
+  runtime_assets:
+  - sensu/http-checks:0.6.0
```

Comment:
gonna need to update this runtime asset version example in the readme to match the version we expect to tag that introduces the new executable.  Small thing, but has to be changed after merge but before tagging

Reply:
looks like this is correct expected version